app/jobs/scrape_job.rb

```diff
@@ -0,0 +1,47 @@
+require 'nokogiri'
+require 'open-uri'
+require 'socket'
+
+class ScrapeJob < ApplicationJob
+  queue_as :default
+
+  def perform
```

Comment:
This looks like the older code. Was the newer code not working?

Reply:
We no longer need the ScrapeJob, so I could remove this file